modules/common/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/common/config/Config.scala

```diff
@@ -245,8 +240,14 @@ object Config {
   implicit val metricsDecoder: Decoder[Metrics] =
     deriveDecoder[Metrics]
 
-  implicit val opsGenieDecoder: Decoder[OpsGenie] =
-    deriveDecoder[OpsGenie]
+  implicit val http4sUriDecoder: Decoder[Uri] =
+    Decoder[String].emap(s => Either.catchOnly[ParseFailure](Uri.unsafeFromString(s)).leftMap(_.toString))
```

Comment:
Why to use `unsafeFromString` and then catch an exception? There's a proper method for doing. You should always avoid `unsafe` methods.